last_goal_effect: true
last_goal_tolerance: 0
canceled_goal_effect: true
canceled_goal_tolerance: 0.5
on_map_branch_detection: true
omb_map_window_x: 1
omb_map_window_y: 1
on_map_branch_rate: 0.9
duplicate_detection: true
duplicate_tolerance: 1.5
log_current_time: 10
newer_duplication_threshold: 10000
on_map_frontier_detection: false
omf_map_window_x: 1
omf_map_window_y: 1
on_map_frontier_rate: 0.91
distance_weight: 2.5
direction_weight: 1.5
variance_threshold: 1.5
covariance_threshold: 0.7
other_robot_weight: 1
duplicate_coeff: 1.2
on_map_coeff: 1.1
